src/components/cc-stretch/cc-stretch.stories.js

```diff
@@ -0,0 +1,100 @@
+// Don't forget to import the component you're presenting!
+import './cc-stretch.js';
+import { makeStory, storyWait } from '../../stories/lib/make-story.js';
+import { enhanceStoriesNames } from '../../stories/lib/story-names.js';
+
+export default {
+  title: '🧬 Atoms/<cc-stretch>',
+  component: 'cc-stretch',
+};
+
+const baseHtml = `
+  <p id="item-1">A short content</p>
+  <p id="item-2">A longer content ?</p>
+  <p id="item-3">The very long content the component bases it's <code>width</code> on.<br>It also bases its <code>height</code> on this content.</p>
+`;
+
+const conf = {
+  component: 'cc-stretch',
+  // language=CSS
+  css: `
+    cc-stretch {
+      border: 0.2em solid red;
+    }
+
+    code {
+      background-color: var(--cc-color-bg-neutral);
+      padding-inline: 0.3em;
+    }
+
+    p {
+      margin: 0;
+    }
+  `,
+};
+
+export const defaultStory = makeStory(conf, {
+  items: [
+    {
+      visibleElementId: 'item-1',
+      innerHTML: baseHtml,
+    },
+  ],
+});
+
+export const contentPositioning = makeStory(conf, {
+  items: [
+    {
+      visibleElementId: 'item-2',
+      innerHTML: baseHtml,
+      style: '--cc-stretch-justify-items: start',
+    },
+    {
+      visibleElementId: 'item-2',
+      innerHTML: baseHtml,
+      style: '--cc-stretch-justify-items: end',
+    },
+    {
+      visibleElementId: 'item-2',
+      innerHTML: baseHtml,
+      style: '--cc-stretch-align-items: start',
+    },
+    {
+      visibleElementId: 'item-2',
+      innerHTML: baseHtml,
+      style: '--cc-stretch-align-items: end',
+    },
+  ],
+});
+
+export const disableStretching = makeStory(conf, {
```

Comment:
Could be nice to have a simulation here so that we really see the advantage of using the `cc-strech` component.

Reply:
By the way, I'm wondering which use case this prop covers.

Reply:
It's very situational and that's why I'm not sure how to showcase it properly in stories / simulations.

In `cc-orga-member-card`, we want the badges to be stretched + centered in desktop:
![image](https://user-images.githubusercontent.com/100240294/196641855-1ec7d950-ed62-4597-8834-a050b2690708.png)

But you don't really want that when your viewport is reduced:
![image](https://user-images.githubusercontent.com/100240294/196642211-e47e8364-da0a-42b5-a105-3589cb0b60af.png)

Here we disable the stretching when the component width is below a specific value, but you might also want to disable the stretching based on other conditions (the element being displayed for instance ?).
At the beginning I used CSS to do this, but it felt like I was bypassing the component behavior just because I knew very well how it worked.
This meant I was adding weird CSS to `cc-orga-member-card`:
```css
:host([w-lt-740]) .status__role-mfa cc-stretch > span:not(.cc-stretch--visible) {
  display: none;
}
```

Reply:
I'll go for a simulation showcasing stretching enabled to stretching disabled.

Tell me if you think this property is not relevant / too specific. Do you think we should handle it with CSS from the parent component when needed ?

Reply:
I understand now the use case so this is perfect like that.